crates/corro-agent/src/agent/run_root.rs

```diff
@@ -0,0 +1,241 @@
+//! Start the root agent tasks
+//!
+//! 1. foo
+//! 2. bar
+//! 3. spawn `super::other_module`
```

Comment:
Haha, what does this mean?

Reply:
Uuuuh :see_no_evil: I think I added that comment very early into the refactor and then kinda forgot about it :joy: